docs/tutorial/README.md

```diff
@@ -0,0 +1,119 @@
+
+Here, you will create a simple Kubernetes cluster with KinD defaults running the following command:
+
+```bash
+kind create cluster
+```
+
+Make sure to install the Dapr Control Plane into the cluster with:
+
+```
+helm repo add dapr https://dapr.github.io/helm-charts/
+helm repo update
+helm upgrade --install dapr dapr/dapr \
+--version=1.13.2 \
+--namespace dapr-system \
+--create-namespace \
+--wait
+```
+
+## Running the Hello example
+
+We will be using the Dapr Statestore API and for that we will install a Redis instance into our cluster using Helm: 
+
+```shell
+helm install redis oci://registry-1.docker.io/bitnamicharts/redis --version 17.11.3 --set "architecture=standalone" --set "master.persistence.size=1Gi"
+```
+
+Once Redis is installed we can deploy our application workloads, including the Statestore component by running: 
+
+```shell
+kubectl apply -f deploy/
+```
+
+This creates a Statestore Dapr Component, a Node application and a Python application. 
+
+If you check the `deploy/node.yaml` and `deploy/python.yaml` files you will notice that both are defining two environment variables: 
+
+```yaml
+        - name: DAPR_HTTP_ENDPOINT
+          value: http://nodeapp-dapr.default.svc.cluster.local:3500
+        - name: DAPR_GRPC_ENDPOINT
+          value: http://nodeapp-dapr.default.svc.cluster.local:50001
+```
+
+These two environment variables let the Dapr SDK know where the Dapr endpoints are hosted (usually for the sidecar these are located on `localhost`).
+
+Because these workloads are not annotated with Dapr annotations, the Dapr Control Plane will not inject the Dapr Sidecar, instead we will create two instances of the Dapr Shared Helm Chart for our services to use.
+
+
+## Creating two Dapr Shared instances for our services
+
+For each application service that needs to talk to the Dapr APIs we need to deploy a new Dapr Shared instance. Each instance have a one to one relationship with Dapr Application Ids. 
+
+Let's create a new Dapr Shared instance for the Node (`nodeapp`) application: 
+
+```sh
+helm install nodeapp-shared oci://docker.io/daprio/dapr-shared-chart --set shared.appId=nodeapp
+```
+
+Notice that the `shared.appId` is the name used for the endpoint variables defined in the previous section. 
+
+Let's do the same for the Python application: 
+
+```sh
+helm install pythonapp-shared oci://docker.io/daprio/dapr-shared-chart --set shared.appId=pythonapp
+```
+
+Once both Dapr Shared instances are up, the application should be able to connect to the Dapr APIs. You can validate this by interacting with the `nodeapp` by running: 
+
+```
+kubectl port-forward service/nodeapp 8080:80
+```
+
+And then sending a request to place a new order: 
+
+```shell
+curl --request POST --data "@sample.json" --header Content-Type:application/json http://localhost:8080/neworder
+```
+
+Validate the order has been persisted: 
+
+```shell
+curl http://localhost:8080/order
+```
+
+Expected Output: 
+```
+{ "orderId": "42" }
+```
+
```

Comment:
I would really encourage adding a K8s command here illustrate where the dapr instances are running. Given these are Dapr deployments and this is a default 3 node Kind cluster, you can show that these instance may be running on different nodes compared to where the app is running. In other words, how do you teach someone that Dapr Shared is installed correctly, or how do I look at a cluster and see that this has dapr shared running on it compared to having side car deployment?

Reply:
Agree here! Daprd is being deployed as a Deployment right? I was unsure if deployment or DS initially.

Maybe just add a screenshot of the Daprd's running on the cluster at least?